pkg/kubeapiserver/authorizer/config.go

```diff
@@ -84,10 +77,10 @@ func (config Config) New() (authorizer.Authorizer, authorizer.RuleResolver, erro
 	superuserAuthorizer := authorizerfactory.NewPrivilegedGroups(user.SystemPrivilegedGroup)
 	authorizers = append(authorizers, superuserAuthorizer)
 
```

Comment:
in a follow-up, we'll want to relocate this construction of an authorizer instance from a configuration object to generic k8s.io/apiserver, so kubeapiserver can call it and inject the constructors for node/rbac/abac authorizers

xref https://github.com/orgs/kubernetes/projects/162/views/1?pane=issue&itemId=37744273